src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/BfdSpec.groovy

```diff
@@ -248,6 +290,24 @@ class BfdSpec extends HealthCheckSpecification {
             assert northbound.getLink(isl.reversed).state == IslChangeType.FAILED
         }
 
+        and: "BFD logical ports are deleted"
+        when: "Get isl source port"
+        def srcSwLogPort = isl.srcPort + bfdLogPortDelta
+        def srcSwitchAddress = northbound.getSwitch(isl.srcSwitch.dpId).address
+        grpc.getSwitchLogicalPortConfig(srcSwitchAddress, srcSwLogPort)
+
+        then: "Human readable error is returned, because it was deleted while removing BFD session."
+        def e = thrown(HttpServerErrorException)  // issue 3754
```

Comment:
please do `#3754` of full ticket url. Searching project by just a number is painful

Reply:
fixed